lib/smart_proxy_ansible/runner/ansible_runner.rb

```diff
@@ -23,6 +23,7 @@ def initialize(input, suspended_action:, id: nil)
         @rex_command = action_input[:remote_execution_command]
         @check_mode = action_input[:check_mode]
         @job_check_mode = action_input[:job_check_mode]
+        @diff_mode = action_input[:diff_mode]
```

Comment:
Looking at the naming: https://github.com/theforeman/smart_proxy_ansible/pull/100 chose `job_diff_mode`. For `check_mode` there's both `check_mode` and `job_check_mode`. d67b3d28f690f8b017ffc15bdf9e538fd7ab380e added that.

Should we support both too here?

Reply:
> We could, but this would require changes to [theforeman/foreman_ansible#749](https://github.com/theforeman/foreman_ansible/pull/749) as well similar to [theforeman/foreman_ansible@76a53e3](https://github.com/theforeman/foreman_ansible/commit/76a53e3563df223e82076bc917a42836d9387819) I guess. Let me know what you prefer.

Looking at https://github.com/theforeman/foreman_ansible/pull/764 and https://github.com/theforeman/smart_proxy_ansible/pull/100 I think no changes are needed here and we just need merge these and mine. There are some text conflicts but these should be easy to fix in a rebase of whatever PR is merged later.

Reply:
I have no problem rebasing my work on top of this PR.